### PR TITLE
[PUBDEV-6351] disable hive import tests on hive3

### PR DESCRIPTION
**Pull request by honzasterba**

### CODE REVIEW

File: scripts/jenkins/groovy/hadoopStage.groovy [hunk 1]
```diff
@@ -60,7 +55,6 @@ private GString getH2OStartupCmd_hadoop(final stageConfig) {
                 -n 1 -mapperXmx 2g -baseport 54445 \\
                 -jks mykeystore.jks \\
                 -notify h2o_one_node -ea -proxy \\
-                -jks mykeystore.jks \\
```
Comment (michalkurka) [inline scripts/jenkins/groovy/hadoopStage.groovy]: Why is this removed?

Comment (michalkurka) [reply scripts/jenkins/groovy/hadoopStage.groovy]: talked in person, @honzasterba please add it back

Comment (honzasterba) [reply scripts/jenkins/groovy/hadoopStage.groovy]: fixed
